.vscode/settings.template.json

```diff
@@ -0,0 +1,15 @@
+{
+  "go.useLanguageServer": true,
+  "go.testFlags": ["-v"],
+  "go.testTimeout": "0s",
+  "go.alternateTools": {
+    "go": "/usr/local/bin/awsgo",
+    "dlv": "/usr/local/bin/awsdlv"
+  },
+  "[yaml]": {
+    "editor.defaultFormatter": "redhat.vscode-yaml",
+    "editor.autoIndent": "advanced"
+  },
+  "yaml.format.enable": true,
+  "editor.formatOnSave": true
+}
```

Comment:
`rehat.vscode-yaml` is in the recommended extensions, about awsgo and awsdlv yes, we should add links to internal docs. we cannot have this working out of Datadog as it would require updating code in `test-infra` to allow passing custom infra configurations (subnets, groups etc)